### PR TITLE
adds library page

### PR DESCRIPTION
**Pull request by Robert27**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
 {
-  "version": "0.4.3",
+  "version": "0.4.4",
   "name": "neuland",
   "main": "expo-router/entry",
   "homepage": "https://github.com/neuland-ingolstadt/neuland.app-native",
   "private": true,
-  "packageManager": "yarn@4.0.1",
```
Comment (BuildmodeOne) [inline package.json:7]: Windows will probably always add this back.

File: src/app/(tabs)/timetable.tsx [hunk 2]
```diff
@@ -78,7 +78,8 @@ export default function TimetableScreen(): JSX.Element {
         try {
             const timetable = await getFriendlyTimetable(today, true)
             setRawTimetable(timetable)
-            setLoadingState(LoadingState.LOADED)
+
+            setLoadingState(LoadingState.ERROR)
```
Comment (BuildmodeOne) [inline src/app/(tabs)/timetable.tsx:82]: Is this correct?

File: src/localization/de/navigation.ts [hunk 3]
```diff
@@ -25,6 +25,8 @@ export default {
         mobility: 'Mobilität',
         grades: 'Noten',
         news: 'THI Neuigkeiten',
+        library: 'Bibliothek',
+        libraryCode: 'Bibliotheks Code',
```
Comment (BuildmodeOne) [inline src/localization/de/navigation.ts:29]: On the student ID it's called "Bibliotheksnummer"

File: src/localization/en/navigation.ts [hunk 4]
```diff
@@ -25,6 +25,8 @@ export default {
         mobility: 'Mobility',
         grades: 'Grades',
         news: 'THI News',
+        library: 'Library',
+        libraryCode: 'Library Code',
```
Comment (BuildmodeOne) [inline src/localization/en/navigation.ts:29]: Would prefer number here as well.

**Review by BuildmodeOne — CHANGES_REQUESTED** [review]
Only a partial review on the train (and smartphone lol)🚂
Will review the rest later :)

File: src/localization/en/common.ts [hunk 5]
```diff
@@ -87,5 +93,31 @@ export default {
...
         },
+        library: {
+            reservations: {
+                title: 'Reservations',
+                seat: 'Seat',
+                id: 'Reservation code',
+                alert: {
+                    title: 'Delete reservation',
+                    message: 'Do you really want to delete this reservation?',
+                    confirm: 'Delete',
```
Comment (BuildmodeOne) [inline src/localization/en/common.ts]: This key can be removed

**Review by BuildmodeOne — DISMISSED** [review]
lgtm

**Review by BuildmodeOne — APPROVED** [review]
lgtm

**Review by Robert27 — COMMENTED** [review]
thanks for the review

**Pull request merged**